f8a_worker/workers/stackaggregator_v2.py

```diff
@@ -159,14 +159,14 @@ def aggregate_stack_data(stack, manifest_file, ecosystem, deps, manifest_file_pa
             "user_stack_info": {
                 "ecosystem": ecosystem,
                 "analyzed_dependencies_count": len(dependencies),
-                "analyzed_dependencies": deps,
-                "unknown_dependencies_count": 0,
+                "analyzed_dependencies": dependencies,
```

Comment:
can you rename it correctly? you renamed `deps` to `dependencies` which might lead to same problem in future.

Thanks, it looks good otherwise

Reply:
I don't see any renaming. Anyway, the commit has been merged as part of #401 so I'm closing this.